openandroidinstaller/assets/configs/sweet.yaml

```
@@ -0,0 +1,57 @@
+metadata:
+  maintainer: BjornW
+  brand: xiaomi
+  device_name: Xiaomi Redmi Note 10 Pro
+  is_ab_device: false
+  device_code: sweet
+  supported_device_codes:
+    - sweet
+    - sweetin
+  untested: false
+  notes:
+    - You should install Android 10 or newer ROM.
```

```suggestion
  notes:
    - You should install Android 10 or newer ROM.
```

`untested: false` is not needed, as it is set by default.